### PR TITLE
Release 3.2.2

### PR DESCRIPTION
**Pull request by Goose-Of-War**

### CODE REVIEW

File: routes/git-hook.js [hunk 1]
```diff
@@ -21,7 +22,7 @@ router.post('/', async (req, res) => {
 	if (branch !== 'dev' && branch !== 'main') {
 		return res.send('Automatic webhook updates are only enabled on dev and main branch');
 	}
-	await Tools.updateCode();
+	console.log(await Tools.updateCode());
```
Comment (nishkalprakash) [inline routes/git-hook.js:25]: this log ok to keep in prod?

Comment (Goose-Of-War) [reply routes/git-hook.js:25]: > this log ok to keep in prod?

For now, ye. Until we find out why git-hook sux

File: templates/newsletters/2023-05-1/2023-05-1.njk
Comment (nishkalprakash) [inline templates/newsletters/2023-05-1/2023-05-1.njk:1]: Is there a crossword in this months newsletters?

Comment (Goose-Of-War) [reply templates/newsletters/2023-05-1/2023-05-1.njk:1]: Nein. Much Schade

**Review by nishkalprakash — APPROVED** [review]

File: templates/newsletters/2023-05-1/2023-05-1.njk
Comment (nishkalprakash) [reply templates/newsletters/2023-05-1/2023-05-1.njk:1]: theres code for that in the njk file

Comment (nishkalprakash) [reply templates/newsletters/2023-05-1/2023-05-1.njk:1]: theres code for that in the njk file

**Pull request merged**